docs/src/components/PositionGrid/PositionGrid.component.tsx

```diff
@@ -0,0 +1,160 @@
+/*
+    Inspired by https://material-ui.com/components/tooltips/#tooltips
+
+    Renders a diagram using a formatted table to illustrate to the user what our position values
+    (top, topStart, bottom, bottemEnd, etc) mean for absolutely positioned elements for components
+    like Tooltip and DropDown.
+*/
+
+// REACT
+import * as React from 'react';
+// VENDOR
+import styled from '@xstyled/styled-components';
+import { th } from '@xstyled/system';
+// ANCHOR & COMPONENTS
+import { ScreenClass, Tooltip } from '@retailmenot/anchor';
+import {
+    breakpoints,
+    BreakpointsType,
+    COMPONENT_MIN_WIDTH,
+    responsiveCSS
+} from '../Utils';
+
+interface StyledPositionGrid {
+    breakpoint?: BreakpointsType;
+}
+
+const StyledPositionGrid = styled('div')<StyledPositionGrid>`
+    text-align: center;
+    padding: 1rem 0;
+    ${({breakpoint}) => responsiveCSS(breakpoint, [breakpoints.xs, breakpoints.sm])`
+        overflow-x: scroll;
+    `}
+`;
+
+const StyledTable = styled('table')`
+    padding: 1rem;
+    background-color: ${th('colors.neutrals.silver.base')};
+    display: inline-block;
+    min-width: ${COMPONENT_MIN_WIDTH};
+    width: auto;
+    margin: 0 auto;
+    border-collapse: collapse;
+
+    td {
+        width: 10rem;
+        padding: 0.75rem 0.25rem;
+        font-family: ${th('typography.fontFamily')};
+        font-size: 1.25rem;
+    }
+`;
+
+const SpanHover = styled('span')`
+    cursor: pointer;
+`;
+
+const TdLeft = styled('td')`
```

Comment:
Wouldn't it be easier to have a single `AlignedTD` that accepted a prop for the position?

Reply:
Or `StyledTD` that also takes in all the style attributes of these TDs